0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Tracking Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Team: (fill me in with the team or teams responsible for this RFC)
+- Keywords: (fill me in with keywords which may be useful when searching for this RFC)
+- Previous RFCs: [#0000](https://github.com/rust-lang/rfcs/pull/0000) (fill me in with RFC numbers for any RFCs this RFC supersedes, deprecates, or extends)
+- Previous discussion: (fill me in with links to previous discussions such as internals.rust-lang.org or Zulip threads, or issues)
```

Comment:
I don't think this should go right up at the top either.

Also, we should take care to avoid creating an undue burden here: we should make it clear that you should include notable discussions you're aware of, but that you don't have to exhaustively list every time the topic came up. (Some topics have come up many dozens of times. A few samples should suffice in such cases.)

Reply:
I'll try and clarify the text. Again the motivation for having it here is so that it is useful for tools, not just for humans to read.

Reply:
See above: which tools and for what use cases?